pkg/controller/machineset/machineset_controller_suite_test.go

```diff
@@ -63,7 +63,23 @@ func TestMachinesetController(t *testing.T) {
 var _ = BeforeSuite(func(ctx SpecContext) {
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "..", "..", "install")},
+
+		CRDInstallOptions: envtest.CRDInstallOptions{
+			Paths: []string{
+				filepath.Join("..", "..", "..", "vendor", "github.com", "openshift", "api", "machine", "v1beta1", "zz_generated.crd-manifests", "0000_10_machine-api_01_machines-CustomNoUpgrade.crd.yaml"),
+				filepath.Join("..", "..", "..", "vendor", "github.com", "openshift", "api", "machine", "v1beta1", "zz_generated.crd-manifests", "0000_10_machine-api_01_machinesets-CustomNoUpgrade.crd.yaml"),
+			},
+		},
+
+		// The CRDs in the CRDDirectoryPaths override the one specified in Paths
+		// above. This breaks tests that rely on parts of the CRD only found in
+		// CustomNoUpgrade. Once the migration APIs land in the default CRD then
+		// this can be uncommented and the Paths above removed.
+
+		//	CRDDirectoryPaths: []string{
+		//		filepath.Join("..", "..", "..", "install"),
+		//	},
+
```

Comment:
?

Reply:
I found that if I left this in, the CRDs synced by the `/hack/crd-sync.sh` script would take priority meaning we wouldnt have the AuthoratitiveAPI field in the status. 

I could not find a way to exclude some files from `CRDDirectoryPaths`